### ISSUE TITLE
Wahoo fit files: missing data definition message for local message number 7

### ISSUE THREAD

**Issue by DieterVanmoortel** [tormoder/fit#63]
Received following error parsing a wahoo .fit file.

```
header decoded: size: 14 | protover: 32 | profver: 2027 | dsize: 337952 | dtype: .FIT | crc: 0x4965
definition message parsed: local: 0 | global: FileId | arch: BigEndian | fields: 5
parsed file_id message: {Activity WahooFitness 31 3107002583 2021-12-11 10:14:48 +0000 UTC 65535 }
definition message 

parsed: local: 0 | global: DeveloperDataId | arch: BigEndian | fields: 2
definition message parsed: local: 0 | global: FieldDescription | arch: BigEndian | fields: 5
definition message parsed: local: 0 | global: FieldDescription | arch: BigEndian | fields: 5
definition message parsed: local: 0 | global: FieldDescription | arch: BigEndian | fields: 5
definition message parsed: local: 0 | global: FieldDescription | arch: BigEndian | fields: 5
definition message parsed: local: 0 | global: FieldDescription | arch: BigEndian | fields: 5
definition message parsed: local: 0 | global: FieldDescription | arch: BigEndian | fields: 5
definition message parsed: local: 0 | global: FieldDescription | arch: BigEndian | fields: 5
definition message parsed: local: 0 | global: Event | arch: BigEndian | fields: 4
parsing time: seconds from device power on
parsing data message: missing data definition message for local message number 7
```

[Archive.zip](https://github.com/tormoder/fit/files/7704043/Archive.zip)

Tried with the java FitCSVTool directly and that worked, but the Local Number seems to be 0 everywhere ( which looks like the reason the decoding actually fails. ) 

In code: (reader.go:213)
`d.defmsgs[dm.localMsgType] = dm`

In this case, dm.localMsgType is always 0.

Tried with other wahoo .fit files, but all have the same outcome.

**Comment by DieterVanmoortel** [comment]
Using the fork from "github.com/beyoung/fit" actually fixes this issue.

**Comment by tormoder** [comment]
It may be that this is fixed now in latest release (v0.13.0) which partially supports Developer data fields.
